pypit/spectrographs/keck_nirspec.py

```diff
@@ -13,20 +13,33 @@
 from astropy.io import fits
 
 from pypit import msgs
+from ..par.pypitpar import DetectorPar
+from . import spectrograph
+from .. import telescopes
+
 from pypit import ardebug as debugger
-from. import spectroclass
 
-class KeckNIRSPECSpectrograph(spectroclass.Spectrograph):
+class KeckNIRSPECSpectrograph(spectrograph.Spectrograph):
     """
     Child to handle Keck/LRIS specific code
     """
-
     def __init__(self):
-
         # Get it started
-        spectroclass.Spectrograph.__init__(self)
+        super(KeckNIRSPECSpectrograph, self).__init__()
         self.spectrograph = 'keck_nirspec'
-
+        self.telescope = telescopes.KeckTelescopePar()
+        self.camera = 'NIRSPEC'
+        self.detector = [
+                # Detector 1
+                DetectorPar(dataext=0, dispaxis=0, xgap=0., ygap=0., ysize=1., platescale=0.193,
```

Comment:
I love all this.  I might ask that you have each `DetectorPar` key/item pair on a 
separate line.  To improve readability.

Reply:
Done.